rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -227,53 +227,17 @@ fn handle_system_tray_event(app: &tauri::AppHandle, event: TrayMenuEvent) -> Res
         .ok_or_else(|| anyhow!("can't get Managed struct from Tauri"))?
         .ctlr_tx;
 
-    // TODO: Just handle these in Controller directly: <https://github.com/firezone/firezone/issues/2983>
-    match event {
-        TrayMenuEvent::About => {
-            let win = app
-                .get_window("about")
-                .ok_or_else(|| anyhow!("getting handle to About window"))?;
-
-            if win.is_visible()? {
-                win.hide()?;
-            } else {
-                win.show()?;
-            }
-        }
-        TrayMenuEvent::CancelSignIn => ctlr_tx.blocking_send(ControllerRequest::CancelSignIn)?,
-        TrayMenuEvent::Resource { id } => {
-            ctlr_tx.blocking_send(ControllerRequest::CopyResource(id))?
-        }
-        TrayMenuEvent::Settings => {
-            let win = app
-                .get_window("settings")
-                .ok_or_else(|| anyhow!("getting handle to Settings window"))?;
-
-            if win.is_visible()? {
-                // If we close the window here, we can't re-open it, we'd have to fully re-create it. Not needed for MVP - We agreed 100 MB is fine for the GUI client.
-                win.hide()?;
-            } else {
-                win.show()?;
-            }
-        }
-        TrayMenuEvent::SignIn => ctlr_tx.blocking_send(ControllerRequest::SignIn)?,
-        TrayMenuEvent::SignOut => ctlr_tx.blocking_send(ControllerRequest::SignOut)?,
-        TrayMenuEvent::Quit => ctlr_tx.blocking_send(ControllerRequest::Quit)?,
-    }
+    ctlr_tx.blocking_send(ControllerRequest::SystemTrayMenu(event))?;
```

Comment:
This variable could be inlined, not sure it adds much value if it is only used once :)